src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,73 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+
+public final class Tokenizer {
```

Comment:
Sorry, I meant add a short description of the class here, and also describe public methods (i.e. next()).